### PR TITLE
refactor(vanilla-extract): Switch to V2 + Vite

### PR DESCRIPTION
**Pull request by machour**

### CODE REVIEW

File: vanilla-extract/app/root.tsx [hunk 1]
```diff
@@ -1,38 +1,31 @@
...
 
-export const links: LinksFunction = () =>
-  cssBundleHref ? [{ rel: "stylesheet", href: cssBundleHref }] : [];
-
-export const meta: MetaFunction = () => ({
-  charset: "utf-8",
-  title: "New Remix App",
-  viewport: "width=device-width,initial-scale=1",
-});
+import "./global.css"; // FIXME
```
Comment (machour) [inline vanilla-extract/app/root.tsx:9]: @markdalgleish I'm a bit confused how to handle this file. Its contents doesn't make it to the build anyway, can you help? 🙏🏼

Comment (markdalgleish) [reply vanilla-extract/app/root.tsx]: It's because `**/*.css.ts` was removed from the `sideEffects` field in `package.json`. I've just pushed an update.

**Review by markdalgleish — APPROVED** [review]
Thanks for updating this!

File: vanilla-extract/app/root.tsx (hunk 1 above)
Comment (machour) [reply vanilla-extract/app/root.tsx]: Wow, I would have never figured this out, thanks!

**Pull request merged**